src/oasis-nexus/generated/api.ts

```diff
@@ -406,7 +406,7 @@ In addition to the existing method names, the following special values are suppo
   - 'evm.Call_no_native': Returns EVM calls that are "not likely to be native transfers".
 
  */
-method?: string[];
```

Comment:
I thing something fishy is going on with Orval here.
Looking at the [underlying API spec change](https://github.com/oasisprotocol/nexus/commit/b3d5447094aeb394671e5a1a8fae22bea5710a88#diff-09939be35d5062a609cabcaf1feaea4ed6abeb77c4bb90f3dc1134a634a9f8a7R57), basically what happened was that something that used to be defined directly as a query parameter, has been extracted and declared separately in the `x-query-parameters` section, and is now referenced from there. But the data type didn't really change, so it should remain an string array. So I'm afraid this is in error in Orval resolving references. I'll check out if there is a new version that handles this better, or at least an upstream error report.

Reply:
... well, there is a new major version of Orval, which generates completely different code, but this specific problem is not resolved.

Reply:
False alarm, in fact there was another update on the Nexus side which triggers this change. We are all good.